bedrock/mozorg/redirects.py

```diff
@@ -572,6 +572,9 @@ def decider(request, **kwargs):
     # bug 1003737
     redirect(r'^impressum/?$', '/about/legal/impressum/'),
 
+    # bug 1236910
+    redirect(r'^support(/.*)?$', 'https://support.mozilla.org/'),
```

Comment:
The URL in question seems to be `https://bedrock-prod-deis.eu-west.moz.works/support/firefox/options`, do we need to be excluding the locale prefix here, or will this still catch and redirect?